src/Nethermind/Nethermind.Evm.Test/InvalidOpcodeTests.cs

```diff
@@ -105,6 +105,20 @@ public class InvalidOpcodeTests : VirtualMachineTestsBase
                                 // }
                                 new Instruction[] { }
                             ))))).ToArray();
+        
+        private static readonly Instruction[] LondonInstructions =
+            FrontierInstructions.Union(
+                HomesteadInstructions.Union(
+                    ByzantiumInstructions.Union(
+                        ConstantinopleFixInstructions.Union(
+                            IstanbulInstructions.Union(
+                                BerlinInstructions.Union(
+                                new Instruction[]
```

Comment:
LondonInstructioms

Reply:
> LondonInstructioms

? :) I don't get it. Have I a typo somewhere?